backend/census_historical_migration/make_submission.py

```diff
@@ -0,0 +1,44 @@
+from .workbooklib.end_to_end_core import run_end_to_end
+from .models import ELECAUDITHEADER
+
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def load_historic_data(audit_year, dbkey):
+    """Create a submission for the given audit year and dbkey"""
+    result = {"success": [], "errors": []}
+    gen = ELECAUDITHEADER.objects.get(AUDITYEAR=audit_year, DBKEY=dbkey)
+    user = create_or_get_user(result, gen)
+    run_end_to_end(user.email, dbkey, audit_year)
+
+    return result
+
+
+def create_or_get_user(result, gen):
+    """Returns the user for the submission if found, or a default user otherwise"""
+    user_email = gen.AUDITEEEMAIL
```

Comment:
We decided to use a default user access for the migration process. We should have an ADR on this soon. For now, you can use [this](https://github.com/GSA-TTS/FAC/blob/main/backend/census_historical_migration/sac_general_lib/sac_creator.py#L59) so we remain consistent. Also instead of `auditee.org` use `fac.gsa.gov`. I have a follow up PR to do the same change on the other side

Reply:
Done!